android/src/main/java/com/tailscale/ipn/ui/view/MainView.kt

```diff
@@ -321,6 +322,7 @@ fun PeerList(
 ) {
   val peerList = viewModel.peers.collectAsState(initial = emptyList<PeerSet>())
   val searchTermStr by viewModel.searchTerm.collectAsState(initial = "")
+  val showNoResults = derivedStateOf { searchTermStr.isNotEmpty() && peerList.value.isEmpty() }
```

Comment:
nit: we're usually calling .value when collect/derive the state.  Then you see a bool elsewhere, and not a State<Boolean>..  It's only used once here, so no big deal, but change it if you have sec.

```
  val showNoResults = derivedStateOf { searchTermStr.isNotEmpty() && peerList.value.isEmpty() }.value
```